src/index.html

```diff
@@ -27,8 +27,8 @@
       a.anchor {
         top: calc(-72.38px - 1.5em) !important;
       }
-      .sidenav {
-        height: calc(100% - 72.38px) !important;
+      .sidenav-body {
+        margin-top: 8.25em !important;
```

Comment:
These css tweaks were to support the latest report structure after upgrading the cucumber-forge-report-generator version.